pkg/helm/chartproxy/testdata/mergedAzureRepoIndex.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: v1
```

Comment:
we do not need this file anymore in this PR?

Reply:
removed